README.md

```diff
@@ -67,6 +67,55 @@ fastify.listen(3000, err => {
   console.log(`server listening on ${fastify.server.address().port}`)
 })
 ```
+## Custom transports
+
+By default, passing an object as options to the plugin will configure nodemailer's main transport (SMTP).
+
+If you need a custom transport, simply initialize the transport, and pass it to the plugin instead of an options object. For example, using the `nodemailer-sparkpost-transport`:
+
+```js
```

Comment:
Can you add a `const fastify = require('fastify')()` line, please. (The example code is invalid without it)